libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java

```diff
@@ -0,0 +1,2289 @@
+
+    private EditorImageSettingsListener mEditorImageSettingsListener;
+    private MediaToolbarAction.MediaToolbarButtonClickListener mMediaToolbarButtonClickListener;
+
+    private Drawable mLoadingImagePlaceholder;
+    private Drawable mLoadingVideoPlaceholder;
+
+    private int maxMediaSize;
+    private int minMediaSize;
+
+    private LiveTextWatcher mTextWatcher = new LiveTextWatcher();
+
+    private WPAndroidGlueCode mWPAndroidGlueCode;
+
+    public GutenbergEditorFragment() {
+        mWPAndroidGlueCode = new WPAndroidGlueCode();
+    }
+
+    public static GutenbergEditorFragment newInstance(String title, String content, boolean isExpanded) {
+        mIsToolbarExpanded = isExpanded;
+        GutenbergEditorFragment fragment = new GutenbergEditorFragment();
+        Bundle args = new Bundle();
+        args.putString(ARG_PARAM_TITLE, title);
+        args.putString(ARG_PARAM_CONTENT, content);
+        fragment.setArguments(args);
+        return fragment;
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        mWPAndroidGlueCode.onCreate(getContext());
+
+        ProfilingUtils.start("Visual Editor Startup");
+        ProfilingUtils.split("EditorFragment.onCreate");
+
+        if (savedInstanceState != null) {
+            mTappedMediaPredicate = savedInstanceState.getParcelable(ATTR_TAPPED_MEDIA_PREDICATE);
+        }
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_gutenberg_editor, container, false);
+
+        mTitle = view.findViewById(R.id.title);
+        mWPAndroidGlueCode.onCreateView(
+                view.findViewById(R.id.gutenberg),
+                new OnMediaLibraryButtonListener() {
+                    @Override public void onMediaLibraryButtonClick() {
+                        onToolbarMediaButtonClicked();
+                    }
+                },
+                getActivity().getApplication(),
+                BuildConfig.DEBUG,
+                BuildConfig.BUILD_GUTENBERG_FROM_SOURCE);
+        mSource = view.findViewById(R.id.source);
+
+        mTitle.addTextChangedListener(mTextWatcher);
+//        mContent.addTextChangedListener(mTextWatcher);
+
+        // Set the default value for max and min picture sizes.
+        maxMediaSize = EditorMediaUtils.getMaximumThumbnailSizeForEditor(getActivity());
+        minMediaSize = DisplayUtils.dpToPx(getActivity(), MIN_BITMAP_DIMENSION_DP);
+//        mContent.setMinImagesWidth(minMediaSize);
+//        mContent.setMaxImagesWidth(maxMediaSize);
+
+        // request dependency injection. Do this after setting min/max dimensions
+        if (getActivity() instanceof EditorFragmentActivity) {
+            ((EditorFragmentActivity) getActivity()).initializeEditorFragment();
+        }
+
+        mTitle.setOnTouchListener(this);
```

Comment:
Good point. We're merely following Aztec's example here of not handling this in total, except I guess the lint warning suppression added [here](https://github.com/wordpress-mobile/WordPress-Android/pull/8701/files#diff-303899d48def98fc26da6b6dd4ad4127R29) and [here](https://github.com/wordpress-mobile/WordPress-Android/pull/8701/files#diff-303899d48def98fc26da6b6dd4ad4127R40).

I guess leaving AS warning us is kinda proper at the moment since there's no decision yet. I'm sure we'll try to properly handle this, but apparently not exactly here in this PR. Hope that's OK.

Reply:
It's fine not to handle it right now, but AS warnings are not that visible. So, I'd suggest adding `@SuppressLint("ClickableViewAccessibility")` at the top of the method and maybe also adding a `// TODO` comment explaining the situation.

If we leave it like this, it's hard to tell that it was a deliberate decision to leave it for later, but adding that suppression and a quick comment should be fairly easy. What do you think?

Reply:
Makes sense to add a TODO comment 👍